### DISCUSSION TITLE
Starting and stopping the generator via MQTT

### DISCUSSION THREAD

**Discussion by acato** [jgyates/genmon#1200] [Ideas]
I could not find a way to do so, but I would love to be able to start and stop the generator via an MQTT topic. The scenario would be to enable a "start generator", "stop generator", and "start and transfer" in a Home Assistant dashboard.

**Comment by jgyates** [comment]
The method to start and stop the generator via MQTT is documented here:

https://github.com/jgyates/genmon/wiki/1----Software-Overview#genmqttpy-optional
